src/view/drag-handle/sensor/create-touch-sensor.js

```diff
@@ -263,8 +263,12 @@ export default (callbacks: Callbacks, getDraggableRef: () => ?HTMLElement): Touc
       return;
     }
 
-    // Not stopping the event as we want to allow force press and other events to occur
-    // Scrolling is stopped by onTouchMove
+    // We need to stop parents from responding to this event - which may cause a double lift
+    // We also need to NOT call event.preventDefault() so as to maintain as much standard
+    // browser interactions as possible.
+    // event.preventDefault() in an onTouchStart blocks almost
+    // every other event including force press
+    event.stopPropagation();
```

Comment:
this is the fix?

Reply:
yep - it stops the event bubbling up.
it is super important that we do not call prevent default here - but stopPropogation is fine